R/azure.R

```diff
@@ -0,0 +1,195 @@
+remove_ws <- function(x) {
+  sapply(x,
+         FUN = function(s) { trimws(gsub("\\s+", " ", s)) },
+         USE.NAMES  =F
```

Comment:
```suggestion
         USE.NAMES = FALSE
```

@behrica can you also go through and make sure all the T/F for TRUE/FALSE are represented as TRUE/FALSE instead of T/F.